updatecli/policies/open-release-pr/updatecli.d/open-release-pr.yaml

```diff
@@ -0,0 +1,166 @@
+          BEGIN { in_package = 0 }
+          /^\[package\]/ { in_package = 1 }
+          in_package && /^version = / { sub(/version = ".*"/, "version = \"" new_version "\"") }
+          /^\[/ && !/^\[package\]/ { in_package = 0 }
+          { print }
+        ' Cargo.toml > Cargo.toml.tmp && mv Cargo.toml.tmp Cargo.toml
+      changedif:
+        kind: file/checksum
+        spec:
+          files:
+            - Cargo.toml
+
+  updateCargoLockVersion:
+    name: update Cargo.lock with new version
+    scmid: "policy-repo"
+    disablesourceinput: true
+    dependson:
+      - condition#checkCargoLockExists
+      - target#updateCargoTomlVersion
+    kind: shell
+    spec:
+      shell: /usr/bin/bash
+      command: |
+        export package_name='{{ source "cargoPackageName" }}'
+        export new_version='{{ source "newestGitHubDraftReleaseVersion" }}'
+        # note: if testing locally, copy the awk command without trailing spaces
+        awk -v new_version="$new_version" -v package_name="$package_name" '
+          BEGIN { package_found = 0 }
+          /^\[\[package\]\]/ { package_found = 0 }
+          $0 ~ "^name = \"" package_name "\"$" { package_found = 1 }
+          package_found && /^\s*version = / { sub(/version = ".*"/, "version = \"" new_version "\"") }
+          { print }
+        ' Cargo.lock > Cargo.lock.tmp && mv Cargo.lock.tmp Cargo.lock
+      changedif:
+        kind: file/checksum
+        spec:
+          files:
+            - Cargo.lock
+
+actions:
+  openUpdatePR:
+    title: 'build: Prepare for release {{ source "newestGitHubDraftReleaseVersion" }}'
+    kind: "github/pullrequest"
+    scmid: "policy-repo"
+    spec:
+      automerge: {{ .pr.automerge }}
+      mergemethod: squash
+      description: |
+        Automatic bump of versions to prepare for a release with version `{{ source "newestGitHubDraftReleaseVersion" }}`.
+        Merging this PR will trigger a release of this policy.
+
+        Changes since last released tag: (if no tag, since HEAD^)
+        https://github.com/{{ .github.owner }}/{{ .github.repo }}/compare/{{ source "newestReleaseTagVersion" | default "HEAD^" }}...updatecli_main_release_pr
+
+        > [!NOTE]
+        > This PR has been created by automation.
+      draft: {{ .pr.draft }}
+#     {{ if .pr.labels }}
+      labels:
+#           {{ range .pr.labels }}
+        - "{{ . }}"
+#           {{ end }}
+#     {{ end }}
+
+scms:
+  policy-repo:
+    kind: github
+    spec:
+      user: "{{ .github.git_author }}"
+      email: "{{ .github.git_email }}"
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repo }}"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ requiredEnv .github.user }}"
+      branch: "{{ .github.branch }}"
+      commitusingapi: true # enable cryptographically signed commits
+      commitmessage:
+        type: "build"
+        title: 'Prepare for release {{ source "newestGitHubDraftReleaseVersion" }}'
```

Comment:
For info, all commits created by this pipeline will have this title.
If unset, Updatecli uses the target name as a commit title which I find cleaner

Reply:
Indeed, applied with https://github.com/kubewarden/automation/pull/99/commits/9707acb3900c29d44ac7977521841821ad46e4b9.